apps/tangle-dapp/containers/TransferTxContainer/TransferTxContainer.tsx

```diff
@@ -221,11 +237,22 @@ const TransferTxContainer: FC<TransferTxContainerProps> = ({
             />
           </BridgeInputGroup>
 
-          {/* TODO: This is temporary, to display the error message if one ocurred during the transaction. */}
+          {isMaxAmount && (
+            <Alert
+              type="warning"
+              size="sm"
+              title="You are sending the maximum amount"
+              description={`Consider keeping a small amount for transaction fees and future txes.`}
```

Comment:
<strong>Warning:</strong> Ensure transaction processes successfully by reserving a portion of funds for transaction fees.

Reply:
title line can be removed; ensure icon is aligned to top. 

<img width="544" alt="Screenshot 2024-04-25 at 9 32 38 PM" src="https://github.com/webb-tools/webb-dapp/assets/109539270/41f171a7-af96-43fd-9e72-642e5748433a">